src/shogun/machine/gp/InferenceMethod.cpp

```diff
@@ -294,4 +297,21 @@ void CInferenceMethod::update_train_kernel()
 	m_ktrtr=m_kernel->get_kernel_matrix();
 }
 
+void CInferenceMethod::set_compute_gradients(bool is_enabled)
```

Comment:
super minor: we usually call the variable in setters the same as the member, but without the "m_" prefix